pytorch_lightning/profiler/pytorch.py

```diff
@@ -323,101 +281,55 @@ def summary(self) -> str:
 
             if self.export_to_chrome:
                 filename = f"{action_name}_{local_rank}_trace.json"
-                path_to_trace = filename if self.path_to_export_trace is None \
-                    else os.path.join(self.path_to_export_trace, filename)
+                path_to_trace = (
+                    filename
+                    if self.path_to_export_trace is None else
+                    os.path.join(self.path_to_export_trace, filename)
+                )
                 function_events.export_chrome_trace(path_to_trace)
 
             if self.emit_nvtx:
-                return output_string
+                return recorded_stats
```

Comment:
Shouldn't this be outside of the `profiled_actions` loop?

Reply:
Yes, could be moved with self.enabled.